### PR TITLE
Co/version policy

### PR DESCRIPTION
**Pull request by cmastr**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4348.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/legal/version-policy/index.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ beta: false
 enterprise: false
 ---
 
-D2iQ&reg; supports N-2 of the latest `MAJOR.MINOR` version of Kommander. For example, if the current version of Kommander&reg; is version 2.0, then D2iQ supports all patch versions of Kommander 2.0, 1.4, and 1.3.
+D2iQ&reg; supports N-2 of the latest `MAJOR.MINOR` version of Kommander. For example, if the current version of Kommander&reg; is version 2.2, then D2iQ supports all patch versions of Kommander 2.2, 2.1, and 1.4.
 
-When the 2.1 version releases, support continues for 2.0, and 1.4. Support for Kommander version 1.3.x expires. Users should upgrade their Kommander clusters with every new release to stay up-to-date with the latest features and bug fixes.
+When the 2.3 version releases, support continues for 2.2, and 2.1. Support for Kommander version 1.4.x expires. Users should upgrade Kommander with every new release to stay up-to-date with the latest features and bug fixes.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/legal/version-policy/index.md:13]:
```suggestion
When the 2.3 version releases, support continues for 2.2, and 2.1. Support for Kommander version 1.4.x expires. Users should upgrade Kommander with every new release to stay up-to-date with the latest features and bug fixes.
```

**Review by Colin1841 — APPROVED** [review]

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/konvoy/2.2/legal/version-policy/index.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ beta: false
 enterprise: false
 ---
 
-D2iQ&reg; supports N-2 of the latest `MAJOR.MINOR` version of Konvoy. For example, if the current GA version of Konvoy&reg; is 2.0, then D2iQ supports all patch versions of Konvoy 2.0, 1.8, and 1.7.
+D2iQ&reg; supports N-2 of the latest `MAJOR.MINOR` version of Konvoy. For example, if the current GA version of Konvoy&reg; is 2.2, then D2iQ supports all patch versions of Konvoy 2.1, and 1.8.
```
Comment (mmcelroy91) [inline pages/dkp/konvoy/2.2/legal/version-policy/index.md:11]:
```suggestion
D2iQ&reg; supports N-2 of the latest `MAJOR.MINOR` version of Konvoy. For example, if the current GA version of Konvoy&reg; is 2.2, then D2iQ supports all patch versions of Konvoy 2.1 and 1.8. 
```

**Review by mmcelroy91 — APPROVED** [review]

**Comment by msdolbey** [comment]
@cmastr - I updated the default version of k8s here - default in 2.2 is kubernetes version 1.22.8

I pushed the change, so you'd have to pull if you have any other changes you need to push from your machine.

**Comment by msdolbey** [comment]
The vale error is unrelated.

**Review by msdolbey — APPROVED** [review]
👍 
Sweet Release

**Pull request merged**